README.md

```diff
@@ -31,26 +31,32 @@ pnpm add svelte-grid-extended
 
 ### Table of Contents
 
-- [Description](#description)
-- [Installation](#installation)
-- [Usage](#usage)
-  - [Basic](#basic)
-  - [Static grid](#static-grid)
-  - [Grid without bounds](#grid-without-bounds)
-  - [Styling](#styling)
-  - [Disable interactions](#disable-interactions)
-  - [Collision Behavior](#collision-behavior)
-    - [None](#none)
-    - [Push](#push)
-    - [Compress](#compress)
-  - [Custom move/resize handle](#custom-moveresize-handle)
-  - [Two way binding](#two-way-binding)
-- [API Documentation](#api-documentation)
-  - [Grid props](#grid-props)
-  - [GridItem props](#griditem-props)
-  - [Style related props:](#style-related-props)
+- [svelte-grid-extended 🍾](#svelte-grid-extended-)
```

Comment:
Your md plugin broke table of contents. `- [svelte-grid-extended 🍾](#svelte-grid-extended-)` is unnecessary part.